### PR TITLE
[CELEBORN-139][BUG] Fix read wrong yaml file format when loading config

### PR DESCRIPTION
**Pull request by nafiyAix**
### What changes were proposed in this pull request?
Fix read wrong yaml file format when loading config

### Why are the changes needed?
its file format should be yaml rather than yml when creating quota config file.

### Does this PR introduce _any_ user-facing change?
yes

### CODE REVIEW

**Comment by AngersZhuuuu** [comment]
Pls change change doc of `QUOTA_CONFIGURATION_PATH`

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala [hunk 1]
```diff
@@ -2362,7 +2362,8 @@ object CelebornConf extends Logging {
     buildConf("celeborn.quota.configuration.path")
       .withAlternative("rss.quota.configuration.path")
       .categories("quota")
-      .doc("Quota configuration file path.")
+      .doc("Quota configuration file path. The file format should be yaml. Quota configuration file template can be " +
+        "found under conf directory.")
```
Comment (AngersZhuuuu) [inline common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:2366]: Also show defauklt path is `$RSS_HOME/quota.yaml`

**Review by AngersZhuuuu — APPROVED** [review]

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks!

**Comment by waitinfuture** [comment]
> Do we need to cherry pick it into 0.2.0? cc: @AngersZhuuuu @waitinfuture

I think so

**Pull request merged**